lib/node/valueExpression.ts

```diff
@@ -71,6 +82,30 @@ const ternaryMethod = (operator: string, separator: string) => {
     };
 };
 
+const prefixUnaryMethod = (operator: string) => {
+    return function(this: INodeable) {
+        return prefixUnaryOperator(operator)(this)
+    };
+}
+
+const postfixUnaryMethod = (operator: string) => {
+    return function(this: INodeable) {
+        return postfixUnaryOperator(operator)(this)
+    };
+}
+
+const binaryMethod = (operator: string) => {
+    return function(this: INodeable, right: any) {
+        return binaryOperator(operator)(this, right);
```

Comment:
This creates a new function every time a binary operator is called which would impact performance if the operator is called in a loop